tests/diff.yml

```diff
@@ -5,8 +5,8 @@ args: ["-timeout=20s", "-shuffle=on", "-race", "./..."]
 results:
   ferretdb:
     stats:
-      expected_fail: 12
-      expected_pass: 5
+      expected_fail: 14
+      expected_pass: 7
```

Comment:
@AlekSi we have one test for insert and one test for update, so we expect two more tests to fail, and two more tests to pass, right? Somehow, dance still fails, and I don't understand why.

Reply:
![image](https://user-images.githubusercontent.com/1684673/196372433-067904e3-318c-4ddd-b6a6-62ca9e71097a.png)